From: Andrew Udvare <audvare@gmail.com>
Subject: [PATCH] makefile: remove ui steps

The ebuild builds the React UI itself, so drop the ui targets from the
release and generate targets.
---
--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@
 endif
 
 .PHONY: release
-release: pre-ui generate ui build-release
+release: generate build-release
 
 # targets to set various build flags
 # use combinations on the make command-line to configure a build, e.g.:
@@ -279,7 +279,7 @@
 
 # Regenerates GraphQL files
 .PHONY: generate
-generate: generate-backend generate-ui
+generate: generate-backend
 
 .PHONY: generate-ui
 generate-ui:
